src/Support/Str.php

```diff
@@ -15,6 +17,11 @@ public static function __callStatic($method, $parameters)
         return call_user_func_array([StaticStringy::class, $method], $parameters);
     }
 
+    public static function ascii($value, $language = 'en')
+    {
+        return ASCII::to_ascii((string) $value, $language, false, Site::current()->transliterateSymbols());
```

Comment:
Is there a reason for using `false` as the third argument? The default is `true`.

https://github.com/voku/portable-ascii/blob/b56450eed252f6801410d810c8e1727224ae0743/src/voku/helper/ASCII.php#L798

Reply:
I must've looked at the wrong parameter